db/migrate/20230822202946_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author_id, foreign_key: { to_table: :users }
+      t.references :post, foreign_key: true
+      t.text :text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] You created this migration file before creating the migration file for the `Post` model. This would cause this error:

![image](https://github.com/toyman640/blog/assets/77106962/990669ea-4d00-41ba-a428-552553261d68)

Please ensure that the migration for creating the `Post` model is done before that of the `Like` and the `Comment` models. 👍 😎